### PR TITLE
correct stack track outputs and error formatting

### PR DESCRIPTION
**Pull request by ryanfkeepers**

### CODE REVIEW

File: err_fmt_test.go [hunk 1]
```diff
@@ -0,0 +1,863 @@
+package clues_test
+
+import (
+	stderr "errors"
+	"fmt"
+	"regexp"
+	"testing"
+
+	"github.com/alcionai/clues"
```
Comment (ashmrtn) [inline err_fmt_test.go:9]: not sure how you want to handle import ordering in this repo, but if the repo imports are supposed to be on their own this should be in a different block

File: err_fmt_test.go [hunk 2]
```diff
@@ -0,0 +1,863 @@
...
+			fmt.Printf("\n-----\n%s\n-----\n", result)
+			fmt.Printf("\n-----\n\"%v\"\n-----\n%#v\n-----", c.reExpect, result)
+			t.Errorf("unexpected format for template %#v\nexpected \"%v\"\ngot %#v", c.tmpl, c.reExpect, result)
+		}
+	})
+}
+
+var globalSentinel = errors.New("sentinel")
+
+func makeOnion(base error, mid, top func(error) error) error {
```
Comment (ashmrtn) [inline err_fmt_test.go:55]: 🧅

**Review by ashmrtn — APPROVED** [review]

File: err_fmt_test.go (hunk 1 above)
Comment (ryanfkeepers) [reply err_fmt_test.go:9]: Ah well, we can come back to it.

File: err_fmt_test.go (hunk 2 above)
Comment (ryanfkeepers) [reply err_fmt_test.go:55]: 🧅 🔪 🥲

Comment (ashmrtn) [reply err_fmt_test.go:55]: that just makes me want to make a `diceOnion()` and integrate it into the test code somehow 🤔

**Pull request merged**